src/store/getters/state-getters.ts

```diff
@@ -927,7 +927,7 @@ export const getCompletingParty = (state: StateIF): CompletingPartyIF => {
 }
 
 /** The Create Special Resolution object. */
-export const getcreateResolution = (state: StateIF): CreateResolutionIF => {
+export const getCreateResolution = (state: StateIF): SpecialResolutionIF => {
   return state.stateModel.createResolution
```

Comment:
Hmm, might be nice to have 1 name for the same thing instead of 3 or 4!

My suggestion: similar to its interface ;)